oak/server/rust/src/lib.rs

```diff
@@ -1,3 +1,21 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Functionality to allow use of Rust code within the Oak Runtime.
+
 // TODO: Plug in memory allocator and depend on https://doc.rust-lang.org/alloc/ to enable use of
 // heap.
 
```

Comment:
Actually I think this can be removed, right?

Reply:
TODONE.